app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -289,14 +294,30 @@ class HomeMenu(
             onItemTapped.invoke(Item.Extensions)
         }
 
-        val syncSignInItem = BrowserMenuImageText(
-            context.getString(R.string.library_synced_tabs),
+        val syncedTabsItem = BrowserMenuImageText(
+            context.getString(R.string.synced_tabs),
             R.drawable.ic_synced_tabs,
             primaryTextColor
         ) {
             onItemTapped.invoke(Item.SyncTabs)
         }
 
+        val syncItemTitle =
+            if (accountManager.accountProfile()?.email != null) {
```

Comment:
I think we want to check with someone understands the FxA code a bit more to check if this is good enough because there are at least 3 states iirc. (Signed in, Need to reconnect, Sign in to sync). If we don't check if the account is authenticated, I think we can run into the 2nd state where the user actually needs to reconnect.